script/Ciel/Day 11/11_01_CIEL11_2_2.txt

```diff
@@ -288,12 +288,12 @@ Now, I must begin the feast, just as I have promised him.
 [offset:29470]{
 -- Page 135, Offset 29470.
 -- 天体の卵。
-The Celestial Egg.
+The %{i}Celestial Egg%{/i}.
```

Comment:
I know we set a standard for italicizing the first instance of lore terms but... I honestly don't like it that much. It kinda ruins some of the mystery / dumbs things down for the reader on some level, and invites awkward questions about what qualifies (e.g, Pantheon later on this page doesn't?). @Petrikow may have thoughts.

Reply:
I guess we can just italicize the terms that have an entry in the glossary? But sure, if you two decided to remove it, I'll have no qualms about it.
Also about the Pantheon thing, I found another instance in an earlier file but forgot to italicize that one.

Reply:
Gotta think what the purpose of it is.

In general it's to signal "you need to remember this because it's something that has a special Nasuverse connotation" 
These do have their connotations, but they're not really terms because they don't get explained.
I.E, yeah stuff like this doesn't need it.
Neither does Pantheon and Original One

Reply:
Alright let's stick to the glossary terms then.